components/feature/pwa/src/main/java/mozilla/components/feature/pwa/WebAppShortcutManager.kt

```diff
@@ -38,21 +38,28 @@ import mozilla.components.feature.pwa.WebAppLauncherActivity.Companion.ACTION_PW
 import mozilla.components.feature.pwa.ext.installableManifest
 
 private val pwaIconMemoryCache = IconMemoryCache()
+
 const val SHORTCUT_CATEGORY = "mozilla.components.pwa.category.SHORTCUT"
 
+/**
+ * Helper to manage pinned shortcuts for websites.
+ *
+ * @param httpClient Fetch client used to load website icons.
+ * @param storage Storage used to save web app manifests to disk.
+ * @param supportWebApps If true, Progressive Web Apps will be pinnable.
+ * If false, all web sites will be bookmark shortcuts even if they have a manifest.
+ */
 class WebAppShortcutManager(
     context: Context,
     httpClient: Client,
     private val storage: ManifestStorage = ManifestStorage(context),
```

Comment:
Should we make this a required parameter without a default value? Since we want to share the storage between components it seems to be an easy mistake now to accidentally have multiple instances with this here creating its own by default.